mteb/evaluation/evaluators/RerankingEvaluator.py

```diff
@@ -224,7 +280,7 @@ def _compute_metrics_instance(self, query_emb, docs_emb, is_relevant):
 
         mrr = self.mrr_at_k_score(is_relevant, pred_scores_argsort, self.mrr_at_k)
         ap = self.ap_score(is_relevant, pred_scores.cpu().tolist())
-        return {"mrr": mrr, "ap": ap}
+        return {"mrr": mrr, "ap": ap}, pred_scores.cpu().tolist()
```

Comment:
We can then not return the scores here, since the calculation will be done in this function.

Reply:
Okay